### PR TITLE
Fix Issue of Escaping Content of Ticket Description

### PR DESCRIPTION
**Pull request by hoseinrafiei**
## Problem this Pull Request solves
Fixes https://github.com/eventespresso/event-espresso-core/issues/3963 issue.

## How has this been tested

Screen Shot 2022-07-22 at 17 14 41

Screen Shot 2022-07-22 at 17 15 26

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,4 @@ vendor/
 *.DS_Store
 *.idea
 *.vscode
+tests/.phpunit.result.cache
```
Comment (tn3rb) [inline .gitignore:19]: ahhh beat me to it... :)

File: caffeinated/admin/new/pricing/espresso_events_Pricing_Hooks.class.php [hunk 2]
```diff
@@ -1539,7 +1539,7 @@ protected function _get_ticket_row(
                     )
                 ),
             'TKT_ID'                        => $default ? 0 : $ticket->ID(),
-            'TKT_description'               => $default ? '' : $ticket->get_f('TKT_description'),
+            'TKT_description'               => $default ? '' : $ticket->get_raw('TKT_description'),
```
Comment (tn3rb) [inline caffeinated/admin/new/pricing/espresso_events_Pricing_Hooks.class.php:1542]: was thinking the issue is escaping on one of the admin page templates that's too strict, 
but if this works, then that's ok with me

**Review by tn3rb — APPROVED** [review]
sure 👍🏻

**Comment by hoseinrafiei** [comment]
@Saam01 

Please test this PR and let us know the results. You should test the issue described here: https://github.com/eventespresso/event-espresso-core/issues/3963

**Comment by Saam01** [comment]
The function seems to be working fine.
image
Output:
image

**Comment by Pebblo** [comment]
@tn3rb & @hoseinrafiei 

Whilst I'm fine with releasing this as is if we have a rich text editor for the description fields should we not be outputting that output in the way the user has it?

Meaning rather than users having to manually add br tags into the content, if they have a new line should we not just output the new line?

Also, should this not be escaped on save? Currently, the values save directly as is in the DB: 
https://monosnap.com/file/vekKpDf9R7dQU9D4SOFybm4Tah31hp

**Comment by tn3rb** [comment]
hey @Pebblo 

> if we have a rich text editor

that's just a plain text area, not an RTE. The new EDTR utilizes an RTE for ticket and datetime descriptions, but not the legacy editor.

> should we not be outputting that output in the way the user has it?

because this is just a plain text editor, it will just display any text as is and can not perform any kind of "rendering"

> should this not be escaped on save

no, escaping is only for display. The ticket and datetime description fields for those models are using the `EE_Post_Content_Field` class which will properly sanitize all content prior to saving

**Comment by Pebblo** [comment]
This changes it back to how it used to work so I think it's fine, I just thought we might be able to include the line breaks but I get it now :)

**Pull request merged**